cmd/kubeadm/app/master/manifests.go

```diff
@@ -544,3 +496,16 @@ func getExtraParameters(overrides map[string]string, defaults map[string]string)
 	}
 	return command
 }
+
+func getImage(image, version string, overrideImage string) string {
+	if overrideImage != "" {
+		return overrideImage
+	}
+	repoPrefix := kubeadmapi.GlobalEnvParams.RepositoryPrefix
+	return map[string]string{
+		etcd:                  fmt.Sprintf("%s/%s-%s:%s", repoPrefix, etcd, runtime.GOARCH, version),
```

Comment:
Doesn't this assume we are using a single repo for all the images?  It's a much more common case, but we should be explicit somewhere in stating you get 1 repo.